CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # [Versions](https://mui.com/versions/)
 
+## v0.0.7
+
+<!-- generated comparing v6.0.0-alpha.2..next -->
+
+_Apr 17, 2024_
+
+A big thanks to the 3 contributors who made this release possible. Here is the highlight of this release ✨:
+
+- 🚀 Updated the sx transformation to also support components created with emotion
+
+### `@pigment-css/react@0.0.7`
+
+- Patch WyW's WeakRef usage (#41909) @DiegoAndai
```

Comment:
Not sure if this belongs here. This was applied as a patch and was applicable to the repo. No Pigment specific change here.

Reply:
Yeah, I will drop it, we will have your fix anyway in the next release 👍